packages/overlay/src/components/Debugger.tsx

```diff
@@ -28,7 +30,7 @@ export default function Debugger({
         }
       }}
     >
-      <div className="sentry-debugger">
+      <div className={classNames('sentry-debugger', fullPage ? 'fullscreen' : '')}>
```

Comment:
Hi @HazAT , i think it would be better to have a unique classname here. Something like sentry-fullscreen or spotlight-fullscreen

Reply:
Just fyi, I did the change but the Shadow DOM we use make it not easily possible to overwrite these classes